config/authorization_definitions/api_entreprise.yml

```
@@ -70,6 +70,10 @@ api_entreprise:
       group: "Informations générales"
       link: "https://entreprise.api.gouv.fr/catalogue/inpi/rne/actes_bilans"
       included: true
+    - name: "Liens capitalistiques - DGFIP | 🔐"
+      group: "Informations générales"
```

```suggestion
      group: "Informations financières"
```
Je pense que c'est plutôt dans infos financières ?

---

Cf https://mattermost.incubateur.net/betagouv/pl/35q1zodbg3rb9y13tqmd5c58zy

---

(J'en sais rien tbh)

---

@DorineLam et @skelz0r , je le mettrai dans "informations générales" au même titre que les bénéficiaires effectifs .